lumicks/pylake/detail/image.py

```diff
@@ -180,9 +180,11 @@ def first_pixel_sample_indices(infowave):
     if infowave.size == 0:
         return 0, 0
 
-    return np.argmax(infowave != InfowaveCode.discard), np.argmax(
-        infowave == InfowaveCode.pixel_boundary
-    )
+    pixel_boundary = np.argmax(infowave == InfowaveCode.pixel_boundary)
+    if infowave[pixel_boundary] != InfowaveCode.pixel_boundary:
+        raise RuntimeError("No completed pixel found in Scan")
```

Comment:
```suggestion
        raise RuntimeError("No completed pixel found in image")
```
maybe don't use "Scan" so as not to confuse with the class